packages/zero-cache/src/services/view-syncer/cvr-store.ts

```diff
@@ -211,7 +212,7 @@ export class CVRStore {
           EXTRACT(EPOCH FROM "ttl") * 1000 AS "ttl",
           "inactivatedAt"
           FROM ${this.#cvr('desires')}
-          WHERE "clientGroupID" = ${id} AND (deleted IS NULL OR deleted = FALSE)`,
+          WHERE "clientGroupID" = ${id}`,
```

Comment:
for line 205: `AND (deleted IS NULL OR deleted = FALSE)` could be `AND deleted IS DISTINCT FROM true`